### ISSUE TITLE
When deleting ResourceGroup and query for the operation status the status is null till completed

### ISSUE THREAD

**Issue by LirazRom** [Azure/azure-sdk-for-java#547]
running this code yeild

```
val resourceManagementClient = ResourceManagementService.create(createConfiguration())
    resourceManagementClient.getResourceGroupsOperations.createOrUpdate("forDelete2", new ResourceGroup("eastus2"))
    val res = resourceManagementClient.getResourceGroupsOperations.beginDeleting("forDelete2")
    for(a <- 1 to 10) {
      val resp = resourceManagementClient.getLongRunningOperationStatus(res.getOperationStatusLink)
      if(resp.getStatus == null) println("null") else println(resp.getStatus.toString)
      Thread.sleep(6000)

    }
```
null
null
null
null
null
null
Succeeded
Succeeded
Succeeded
Succeeded

**Comment by devigned** [comment]
Interesting... we'll verify what the api is returning, but I'm guessing that the operation happens so quickly (after it starts), that any intermediate steps between started and succeeded are done within the 6 seconds.

**Comment by jianghaolu** [comment]
This should be fixed in 1.0.0-beta1 & 1.0.0-beta2. Please verify.
